stability-checker/deploy/chart/stability-checker/templates/role.yaml

```diff
@@ -11,7 +11,7 @@ rules:
 
 - apiGroups: [""]
   resources: ["configmaps"]
-  verbs: ["get", "list"]
+  verbs: ["get", "list", "create", "delete", "update"]
```

Comment:
please check if you could use here
```
  resourceNames:
    - <our_cfg_map_name>
```

so the `"create", "delete", "update"` action will be scoped only for given resource. It will be more secure